prusti-tests/tests/compiletest.rs

```diff
@@ -107,6 +107,10 @@ fn run_prusti_tests(group_name: &str, filter: &Option<String>, rustc_flags: Opti
         config.src_base = path;
         run_tests(&config);
     }
+
+    // Delete the nll-facts directory to avoid running out of hard drive
+    // space.
+    std::fs::remove_dir_all("nll-facts").unwrap();
```

Comment:
I think the default location is `log/nll-facts`.

Reply:
Thanks! Let's see if this helps.